### PR TITLE
Feature/detailspage

### PR DESCRIPTION
**Pull request by andy-schwedler**
Thanks for your time!

### CODE REVIEW

File: components/EventCardDetails.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      </StyledLocationLink>
+      <p>
+        more text more text more text more text more text more text more text
+        more text more text more text more text more text more text more text
+        more text more text more text more text more text more text more text
+        more text more text more text more text more text more text more text
+        more text more text more text more text more text more text more text
+        more text more text more text more text more text more text more text
+        more text more text more text more text more text more text more text
+        more text more text
```
Comment (MerleMak) [inline components/EventCardDetails.js]: 😂

File: components/EventCard.js [hunk 2]
```diff
@@ -9,10 +10,25 @@ export default function EventCard({ sampleEvent, onToggleFavorite }) {
...
+// <article>
+//         <BookmarkButton
+//           isFavorite={sampleEvent.isFavorite}
+//           id={sampleEvent.id}
+//           onToggleFavorite={onToggleFavorite}
+//         />
+//         <h3>{sampleEvent.date}</h3>
+//         <p>{sampleEvent.name}</p>
+//         <p>{sampleEvent.category}</p>
+//       </article>
```
Comment (MerleMak) [inline components/EventCard.js]: Please delete unused code ☺️

File: components/EventCardDetails.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import { StyledLocationLink } from "./GlobalStyles";
+
+export default function EventCardDetails({ currentEvent, onToggleFavorite }) {
+  return (
+    <article>
+      {/* <BookmarkButton
+        isFavorite={currentEvent.isFavorite}
+        id={currentEvent.id}
+        onToggleFavorite={onToggleFavorite}
+      /> */}
```
Comment (MerleMak) [inline components/EventCardDetails.js]: Please delete unused code before commiting :)

**Review by MerleMak — COMMENTED** [review]
Good Job!! I left some tiny remarks, but keep up the good work ☺️

File: components/EventCard.js (hunk 2 above)
Comment (andy-schwedler) [reply components/EventCard.js]: Will do!

File: components/EventCardDetails.js (hunk 3 above)
Comment (andy-schwedler) [reply components/EventCardDetails.js]: Will do!

File: components/EventCardDetails.js (hunk 1 above)
Comment (andy-schwedler) [reply components/EventCardDetails.js]: Will delete. Thanks!

**Pull request merged**